code/datums/jobs/job/shipside.dm

```diff
@@ -108,6 +108,11 @@ Godspeed, captain! And remember, you are not above the law."})
 	minimal_access = ALL_MARINE_ACCESS
 	display_order = JOB_DISPLAY_ORDER_EXECUTIVE_OFFICER
 	outfit = /datum/outfit/job/command/fieldcommander
+	multiple_outfits = TRUE
+	outfits = list(
+		/datum/outfit/job/command/fieldcommander,
+		/datum/outfit/job/command/fieldcommander_robot,
```

Comment:
why are you using defines instead of an assoc list or something

Reply:
Also those defines are literally just bootleg isrobots

Reply:
Because assoc is cringe and would involve modifying every single job in the game with more than one outfit, even though 99% of them don't care about this species stuff.
Define so its not snowflake checks - i.e. if new species are added no new code is needed here.